src/pudl/etl/glue_assets.py

```diff
@@ -46,9 +41,9 @@ def create_glue_tables(context):
     # Ensure they are sorted so they match up with the asset outs
     glue_dfs = dict(sorted(glue_dfs.items()))
 
-    return (
-        Output(output_name=table_name, value=df) for table_name, df in glue_dfs.items()
-    )
+    for table_name, df in glue_dfs.items():
```

Comment:
Non-blocking - would a generator expression work here instead? 

```python
return (
    Output(output_name=table_name, value=df)
    for table_name, df in glue_dfs.items()
    if table_name in context.selected_output_names
)
```
? I guess that's not really much better, is it.

Reply:
I'm not sure. I am just working off of the [dagster docs examples](https://docs.dagster.io/concepts/assets/multi-assets#subsetting-multi-assets). IDK if dagster cares if its a return or yield statement.